st2actions/st2actions/runners/fabricrunner.py

```diff
@@ -114,9 +114,16 @@ def pre_run(self):
 
     def run(self, action_parameters):
         LOG.debug('    action_parameters = %s', action_parameters)
-        remote_action = self._get_fabric_remote_action(action_parameters) \
```

Comment:
This was just a code smell, we should select a method based on the runner type and not the presence or value of the entry_point.